src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/CpsPackageReferenceProject.cs

```diff
@@ -30,11 +30,13 @@
 namespace NuGet.PackageManagement.VisualStudio
 {
     /// <summary>
-    /// Represents a project object associated with new VS "15" CPS project with package references.
+    /// Represents a project object associated with Common Project System (CPS) project that has opt'd
+    /// into package references. This includes, but may not be limited to, .NET Project System,
+    /// C++ (with PackageReference support) and MSIX deployment projects.
```

Comment:
```suggestion
    /// Managed C++ (with PackageReference support) and MSIX deployment projects.
```

I think we should be clear that native C++ projects are not getting PackageReference support. Maybe "Managed C++" is not the right name, but I don't know what the right name for "C++ that gets compiled into .NET assemblies" is. I've heard of c++/cx and c++/clr, but don't know what the differences are.

Reply:
What is the difference? Managed C++ can also use cpp and other various resources, you don’t support it in contentfiles? Why is it so complicated to support things that can be supported by just one move? Also deliberately divided into Managed C++ and Native C++.

Reply:
Please don't artificially increase this fragmentation, okay? Our users can accept that the function is not complete at the beginning. It has been almost a year since I submitted this PR, right? We have been waiting for too long.

Reply:
@zivkan This is _C++/CLI_ (v2 of C++ and .NET) not _Managed C++_ (v1 of C++ and .NET)